test/e2e/inference_with_adapters_test.go

```diff
@@ -158,6 +159,47 @@ func validateAdapterAdded(workspaceObj *kaitov1alpha1.Workspace, deploymentName
 	})
 }
 
+func validateAdapterLoadedInVLLM(workspaceObj *kaitov1alpha1.Workspace, deploymentName string, adapterName string) {
+	execOption := corev1.PodExecOptions{
+		Command:   []string{"bash", "-c", "apt-get update && apt-get install curl -y; curl -s 127.0.0.1:5000/v1/models | grep " + adapterName},
```

Comment:
"curl -s 127.0.0.1:5000/v1/models"

What does this return?

Reply:
```
{
   "object":"list",
   "data":[
      {
         "id":"falcon-7b",
         "object":"model",
         "created":1735276136,
         "owned_by":"vllm",
         "root":"falcon-7b",
         "parent":"None",
         "max_model_len":1024
      },
      {
         "id":"e2e-adapter2",
         "object":"model",
         "created":1735276136,
         "owned_by":"vllm",
         "root":"/xxxxx",
         "parent":"falcon-7b",
         "max_model_len":"None"
      }
   ]
}
```

like this. the adapter's parent is the original model.